assets/upgradePatches.json

```diff
@@ -162,6 +162,18 @@
         "name": "mtqs.mtq.kubevirt.io",
         "namespace": ""
       }
+    },
+    {
+      "semverRange": "<1.14.0",
+      "groupVersionKind": {
+        "group": "",
+        "version": "v1",
+        "kind": "ConfigMap"
+      },
+      "objectKey": {
+        "name": "kubevirt-ui-features",
+        "namespace": "default"
+      }
```

Comment:
why do we need this?

Reply:
on the other hand, I see I forgot to add the removal of the second CM (`kubevirt-user-settings`) and their Roles and RoleBindings. 6 objects in total. Do you think we should remove all of them? or just leave them as is?

Reply:
oh, I missed the `objectsToBeRemoved` - it was out  side of my screen... So yes, please remove everything.